config/application.php

```diff
@@ -36,12 +36,24 @@
         $dotenv->required(['DB_NAME', 'DB_USER', 'DB_PASSWORD']);
     }
 }
+/**
+ * Set up Environment Types
+ * Default: production, staging, development
+ */
+Config::define('WP_ENVIRONMENT_TYPES', env('WP_ENVIRONMENT_TYPES') ? explode(',', env('WP_ENVIRONMENT_TYPES')) : ['production', 'staging', 'development']);
```

Comment:
As of WP 5.5.1 `WP_ENVIRONMENT_TYPES` [is deprecated](https://make.wordpress.org/core/2020/08/27/wordpress-environment-types/). Very short life...

The following environment types will be supported as of 5.5.1:

```
-     production – this is the default. A site that is running live, connected to the internet and reachable on the internet.
-     staging – this is what you would use for staging environments, probably both connected to and reachable on the internet.
-     development – this is what you would use for development environments that are reachable on the internet, we automatically enable WP_DEBUG on environments where this is the environment type.
-     local – added in 5.5.1, this (usually development) environment can reach the internet but is not be reachable from the internet.
```

As these include Bedrock defaults ['production', 'staging', 'development'], setting `WP_ENVIRONMENT_TYPES` is obsulete. 

```suggestion
```